### DISCUSSION TITLE
Custom Sky Cultures

### DISCUSSION THREAD

**Discussion by hmkp** [Stellarium/stellarium#2747] [Q&A]
Looking to create constellation lines as well as names.  I have minimal experience with coding though.

**Comment by alex-w** [comment]
Please check Stellarium User Guide - we have one chapter with description about files formats and "how to do it" for sky cultures.

**Comment by Mdlalose** [comment]
Dear Stellarium Team,

My name is Dr. Mthokozisi Mdlalose, an Astrophysicist at the University of KwaZulu-Natal, South Africa. I've been using Stellarium for my star gazing for at least two years, and I find it even more impressive once you upgrade. Thank you for these fantastic tools.

I saw a feature on Sky culture and was inspired to lead a project on adding the isiZulu language as one option in Stellarium. IsiZulu is a sub-Bantu language, and it is spoken across Namibia, Zambia, Lesotho, Eswatini, Zimbabwe, Botswana, and Mozambique and it is related to Kiswahili. It has reached the history of Astronomy, evidenced through Sanusi Credo Mutwa books, Songs of the Stars, and Indaba My children.

Could you guide me on how I can initiate such a project? I can code in python, and I am willing to learn if any new skill is required to accomplish this task.

I appreciate any help you can provide.

Dr, Mthokozisi Mdlalose, 

MdlaloseM2@ukzn.ac.za

**Reply by alex-w** [reply]
Dear Dr. Mthokozisi Mdlalose,

Probably (according to the text) you want to get support for the Zulu language in Stellarium. So, for adding support the language no need programming skills - just register at transifex (https://www.transifex.com/stellarium/stellarium/), make request to adding support some language (Zulu (zu) in your case) and make request to joining to group of translators of requested language.

**Comment by Mdlalose** [comment]
Thanks Alex,

I just do that, and update you if there are any challenges.

Kind regards,

Mthokozisi



On Wed, 9 Nov 2022 at 09:27, Alexander V. Wolf ***@***.***>
wrote:

> Dear Dr. Mthokozisi Mdlalose,
>
> Probably (according to the text) you want to get support Zulu language in
> Stellarium. So, for adding support the language no need programming skills
> - just register at transifex (
> https://www.transifex.com/stellarium/stellarium/), make request to adding
> support some language (Zulu (zu) in your case) and make request to joining
> to group of translators of requested language.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Stellarium/stellarium/discussions/2747#discussioncomment-4094057>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADDW4VK3EPD3WJJY24TUAY3WHNG6XANCNFSM6AAAAAARCAQGN4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

Dr. Mthokozisi Sibusiso Mdlalose
 Astronomy and Astrophysics, School of Chemistry and Physics.
University of KwaZulu-Natal

Email Address : ***@***.*** ***@***.***>
Cell Phone : (+27) 63 4381 385
https://www.linkedin.com/profile/preview?locale=en_US&trk=prof-0-sb-preview-primary-button
" Character is like pregnancy. You can't hide it"

**Reply by alex-w** [reply]
I've added Zulu (zu) language and sent invitation for you
